test/dataset/test_vad_dataset.py

```diff
@@ -0,0 +1,41 @@
+from math import isclose
+import pytest
+
+import torch
+
+from lhotse.utils import time_diff_to_num_frames
+from lhotse.cut import CutSet
+from lhotse.dataset.vad import VadDataset
+
+
+@pytest.fixture
+def cut_set():
+    return CutSet.from_yaml('test/fixtures/ami/cuts.yml')
+
+
+def test_unsupervised_dataset(cut_set):
+    duration = 5.0
+    feature_threhold = -8.0
+
+    cuts = list(cut_set.cuts.values())
+    assert duration < cuts[0].duration < 2 * duration
```

Comment:
I don't understand the intention of this assertion - why is it testing for duration of range [5, 10]?

Reply:
Eh, it was just for confirming the duration is 6s. Not quite useful indeed. I'll remove this line.